index.md

```diff
@@ -5,10 +5,13 @@ version: develop
 latest: true
 ---
 
-This pipeline can be used to process germline-DNA data, starting from FastQ
+This pipeline can be used to process germline-DNA data, starting with FastQ
```

Comment:
This pipeline can be used to process germline-DNA data with read length above 70bps, ...

"If only bwa-mem is supported now"